### ISSUE TITLE
Jagged edges on text 

### ISSUE THREAD

**Issue by davekiss** [mifi/editly#84]
Hi, super cool project, looking forward to digging in more!

I tried adding a title clip, and it renders out to the video successfully, but the text edges aren't very smooth. Am I doing something wrong, or is this to be expected? Is there a recommended way around this?

Here are my clip settings:

```
    {
      duration: 2,
      layers: [{ type: "title", text: "I'm sorta jaggy", zoomDirection: null }],
    },
```

![image](https://user-images.githubusercontent.com/1256071/96525484-15a54f80-1249-11eb-90c1-3fd44f6ad24f.png)

**Comment by mifi** [comment]
Ohh yea I know this. This is because you don't have any background. It's an issue with fabric.js I think.
Try to add another layer before the text: `{ type: 'fill-color', color: '#000000' }`

I think we could change editly so it automatically adds such a layer to the bottom of the stack, but it will cause a small performance hit.

**Comment by davekiss** [comment]
@mifi you're right, that fixes the issue! Thanks for your quick reply. I'll leave this open in case you wish to track for the auto-add layer idea you have
